partner_identification/models/res_partner_id_number.py

```diff
@@ -22,8 +22,7 @@ def validate_id_number(self):
     name = fields.Char(
         string="ID Number",
         required=True,
-        help="The ID itself. For example, Driver License number of this "
-        "person",
+        help="The ID itself. For example, Driver License number of this " "person",
```

Comment:
You might merge the two strings (remove the double doublequotes between this and person)

Reply:
done! :)